osgar/drivers/usbcam.py

```diff
@@ -57,4 +57,9 @@ def run_input(self):
     def request_stop(self):
         self.bus.shutdown()
 
+
+if __name__ == "__main__":
+    from pprint import pprint
+    pprint(uvc.device_list())
```

Comment:
this is good - I would maybe add this print also when the port is not found

Reply:
done